### DISCUSSION TITLE
Can I help?

### DISCUSSION THREAD

**Discussion by oldenough** [VFPC/VFPC#131] [General]
I can't code in any language that is useful today, but I could help maintain datasets if that would relive you from some of the drudgery?  Janet

**Comment by lennycolton** [comment]
Hi Janet,

The ingest process is about 80% automatic but it can occasionally throw a spanner in the works (and also we forget because organisation isn't our strong suit). We've got a tool to parse the SRD but (as I'm sure you can imagine) it requires a lot of extra stuff to be fed in as well (exit point level rules, SID restrictions, SRD notes, etc.) which is usually where we run into issues.

If you're still interested in helping out with the data sets, I can promise you that ~~it's an extremely boring job which Tom never wants to do~~ it would be a massive help to us. Please feel free to drop me a DM on Discord and we can get you set up.

Cheers,
Lenny
